lib/configuration.rb

```diff
@@ -27,6 +29,8 @@ def initialize
       @dumps_directory = File.join(Dir.pwd, "dumps")
       @database_config_file = File.join(Dir.pwd, "config", "database.yml")
       @container_name = nil
+      @local_mysql_cli_path = nil
+      @local_psql_cli_path = nil
```

Comment:
Couldn't this be just one setting? We don't support mixed dbms within one project anyway, so this could just be the `local_cli_path`.

Reply:
Hmm. Yes. I guess. Feels a bit weird to me to have completely different binaries in the same setting, but I see your point. I'll change it.